README.md

```diff
@@ -501,6 +501,14 @@ Under these levels, the logged messages may include:
 
 The `Debug` and `Trace` loglevels should not be used in production. Under the `Trace` level all packets are logged. This will expose sensitive data related to the SSH connection and the application itself.
 
+## Examples
```

Comment:
You can move this up so it comes right after _Getting Started_.

Reply:
Will do thanks